### DISCUSSION TITLE
How can I integrate pg_squeeze with spilo postgres image?

### DISCUSSION THREAD

**Discussion by suyashsikarwar** [zalando/postgres-operator#1709] [Q&A]
How can I integrate pg_squeeze with spilo postgres image? 

We have a requirement where we need to have pg_squeeze added as an extention to zalando postgres image of v13. How can we do achieve that?

**Comment by tpo** [comment]
one way to do it:
- you take the spilo container
- you extend it with pg_squeeze
- you replace `docker_image` in the [ConfigMap](https://github.com/zalando/postgres-operator/blob/master/manifests/configmap.yaml) with your own extended version of the spilo container
- you deploy the changed `ConfigMap`
- you add an environment variable to the [postgres-operator Deployment](https://github.com/zalando/postgres-operator/blob/master/manifests/postgres-operator.yaml) that you bump everytime you update the `ConfigMap`. The postgres-operator controller will note this and reload the `ConfigMap`.
